postgres/osmap.yaml

```diff
@@ -9,7 +9,3 @@ Fedora:
     baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/fedora/fedora-$releasever-$basearch'
   remove:
     releases: ['9.4', '9.5', '9.6', '10']
-
-Amazon:
-  pkg_repo:
-    baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/redhat/rhel-7-$basearch'
```

Comment:
Moved this block to [`codenamemap.yaml`](https://github.com/saltstack-formulas/postgres-formula/pull/315/files#r665291631).